provider/datagen/src/bin/datagen.rs

```diff
@@ -29,11 +27,11 @@ fn main() -> eyre::Result<()> {
             Arg::with_name("FORMAT")
                 .long("format")
                 .takes_value(true)
+                .required(true)
```

Comment:
Thought/Issue: This is a breaking change, although I support it. It might be safer to leave `.required(true)` as a 2.0-breaking thing. We can still document that you really should be setting your `--format`.

Reply:
I don't think binary CLI arguments are under semver. Clients install a specific `icu4x-datagen`, it's not that it's pulled in as a transitive dependency.

Reply:
They might have a build step that downloads the latest 1.x release and then invokes it.